### PR TITLE
Express model subgenerator and test added

### PR DESCRIPTION
**Pull request by UndeadBaneGitHub**

### CODE REVIEW

File: express-model/templates/_.server.model.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+/**
+ * <%= classifiedModelName %> Schema
+ */
+var <%= classifiedModelName %>Schema = new Schema({
+	// <%= classifiedModelName %> model fields   
+	// ...
+});
+
+mongoose.model('<%= classifiedModelName %>', <%= classifiedModelName %>Schema);
```
Comment (codydaig) [inline express-model/templates/_.server.model.js]: New Line.

File: express-model/templates/_.server.model.tests.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var should = require('should'),
+	mongoose = require('mongoose'),
```
Comment (codydaig) [inline express-model/templates/_.server.model.tests.js]: Spacing should be 2 spaces.

File: express-model/templates/_.server.model.tests.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+		});
+	});
+
+	afterEach(function(done) { 
+		<%= classifiedModelName %>.remove().exec();
+		User.remove().exec();
+
+		done();
+	});
+});
```
Comment (codydaig) [inline express-model/templates/_.server.model.tests.js]: New Line.

**Comment by codydaig** [comment]
Left some line comments.
Don't forget to squash. 
Otherwise, LGTM.

File: package.json [hunk 4]
```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "bluebird": "~3.0.6",
     "chalk": "~1.1.1",
+    "underscore.inflections": "^0.2.1",
```
Comment (ilanbiala) [inline package.json]: ~

File: test/express-model.test.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var path = require('path'),
+  helpers = require('yeoman-generator').test,
+  assert = require('yeoman-generator').assert,
+  temp = require('temp').track();
+
+describe('Express Sub Generators Tests', function () {
+  this.timeout(0);
```
Comment (ilanbiala) [inline test/express-model.test.js]: ?

**Comment by codydaig** [comment]
@ilanbiala LGTY?

**Comment by ilanbiala** [comment]
LGTM

**Pull request merged**